overview/content-types/demo-instructions.nix

```
@@ -136,11 +137,23 @@ in
                 <strong>Build and run a virtual machine</strong>
                 ${config.build-instructions}
               </li>
-              ${lib.optionalString (with self.demo; description == null || description == "") ''
-                <li><span class="option-alert">Missing</span>
-                  <a href="https://github.com/ngi-nix/ngipkgs/blob/main/CONTRIBUTING.md">Contribute usage instructions.</a>
-                </li>
-              ''}
+              ${
+                if (with self.demo; description == null || description == "") then
+                  ''
+                    <li><span class="option-alert">Missing</span>
+                      <a href="https://github.com/ngi-nix/ngipkgs/blob/main/CONTRIBUTING.md">Contribute usage instructions.</a>
+                    </li>
+                  ''
+                else
+                  ''
+                    <li>
+                      <strong>Usage Instructions</strong>
+                    <code>
+                      ${markdownToHtml self.demo.description}
+                    </code>
```

```suggestion
                      ${markdownToHtml self.demo.description}
```

Why rendering it as code? This should simply be text